src/luaotfload-database.lua

```diff
@@ -171,9 +171,8 @@ local stringsub                = string.sub
 local stringupper              = string.upper
 local tableconcat              = table.concat
 local tablesort                = table.sort
-local utf8gsub                 = unicode.utf8.gsub
-local utf8lower                = unicode.utf8.lower
-local utf8len                  = unicode.utf8.len
+local utf8len                  = utf8.len
+local utf8offset               = utf8.offset
```

Comment:
Using `utf8` from the Lua standard library is more portable and does not depend on unsupported third-party libraries.